src/app/app-routing.module.ts

```diff
@@ -1,10 +1,22 @@
 import { NgModule } from '@angular/core';
```

Comment:
Could you please make sure that all TS Lint issues are fixed in your project?

Reply:
![image](https://user-images.githubusercontent.com/53089462/100341510-51200000-2fe5-11eb-826a-974475be10a3.png)